devops/charts/controller/values.yaml

```diff
@@ -14,7 +14,7 @@ image:
   registry: ghcr.io
   repository: bcgov/mobile-attestation-vc-controller/controller
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "cef39c9"
+  tag: "a97d583"
```

Comment:
Intentional?

Reply:
Yes. That is the tag of the newly minted docker container. It changes with every meaningful merge.